Source/WebKit/UIProcess/API/glib/WebKitURISchemeRequest.cpp

```diff
@@ -204,6 +206,26 @@ SoupMessageHeaders* webkit_uri_scheme_request_get_http_headers(WebKitURISchemeRe
     return request->priv->headers.get();
 }
 
+/**
+ * webkit_uri_scheme_request_get_http_body:
+ * @request: a #WebKitURISchemeRequest
+ *
+ * Get the http body of the request.
+ *
+ * Returns: (transfer none): the http body of the @request.
```

Comment:
I would remove “http” from the documentation. URI scheme requests don't involve the HTTP protocol after all.

```suggestion
 * Get the request body.
 *
 * Returns: (transfer none): the body of the @request.
```